lib/core/utils.toit

```diff
@@ -39,8 +39,8 @@ For floats, two NaN's (not-a-number) are identical when they have the same
   `==` where NaN's are never equal, so `float.NAN == float.NAN` is always
   false.
 */
-identical x y:
-  #primitive.core.identical
+//identical x y:
```

Comment:
Not quite sure what to do with this. In particular, it would be nice to keep the documentation around somehow. Maybe keep it around and let it throw? I'll have to change the builtin lookup rules for that to work (they don't find the builtin, if the identifier resolves in the scope).

Reply:
Fixed.